packages/ActionBar/src/components/Filter/FilterItem.js

```diff
@@ -127,6 +139,21 @@ function FilterItem(props) {
             parsingFormat={columns.find(({ id }) => id === selectedColumnId).momentParsingFormat}
           />
         );
+
+      case columnTypes.SINGLE_SELECT:
+        return (
+          <InlineSelect
+            value={`${value}`}
+            onChange={handleChangeSingleSelectFilterValue}
+            selectedLabel={I18n.t(`actionBar.filter.rules.${value}`)}
```

Comment:
`selectedLabel` here should be the active item you see on the UI, for example, by default it's `striker ` , once you choose `midfielder `, it changes to `midfielder `
This is for making the `select` element look like inline expandable